cartridges/LINK_adyen/cartridge/controllers/Adyen.js

```diff
@@ -170,6 +171,29 @@ server.get('GetPaymentMethods', server.middleware.https, function (req, res, nex
   return next();
 });
 
+/**
+ * Get OriginKey for Secured Fields
+ */
+server.get('GetConfigSecuredFields', server.middleware.https, function (req, res, next) {
+    var adyenHelper = require('*/cartridge/scripts/util/AdyenHelper');
+    var	adyenGetOriginKey = require('*/cartridge/scripts/adyenGetOriginKey');
+    var baseUrl = req.querystring.protocol + "//" + Site.getCurrent().getHttpsHostName();
+    var originKey;
+    var loadingContext = "";
+
+    try {
+        originKey = adyenGetOriginKey.getOriginKey(baseUrl).originKeys;
+        loadingContext = adyenHelper.getLoadingContext();
+    } catch (err) {
+        originKey = [];
```

Comment:
do we log the errors in the system? If we do then we could do it here with the caught exception